stacks/go/2.0.0/devfile.yaml

```diff
@@ -0,0 +1,79 @@
+schemaVersion: 2.2.0
+metadata:
+  description: 'Go is an open source programming language that makes it easy to build simple, reliable, and efficient software.'
+  displayName: Go Runtime
+  icon: https://raw.githubusercontent.com/devfile-samples/devfile-stack-icons/main/golang.svg
+  name: go
+  projectType: Go
+  provider: Red Hat
+  language: Go
+  tags:
+    - Go
+  version: 2.0.0
```

Comment:
how come this version is `2.0.0`. Should it be `1.0.2`? I see this for some other stacks too.

Reply:
oops nevermind, i saw the wrong file. Pls ignore.